Cake.XComponent.Test/XcBuildAliasesTest.cs

```diff
@@ -22,14 +22,37 @@ public void SetUp()
         }
 
         [TestFixtureTearDown]
-        public void TearDown()
+        public void TearDoTestFixtureTearDownwn()
         {
             if (Directory.Exists(_xcBuildDirectory))
             {
                 Directory.Delete(_xcBuildDirectory, true);
             }
         }
 
+        [TearDown]
+        public void TearDown()
+        {
+            XcBuild.XcBuildPath = null;
+        }
+
+        [TestCase("xcbuild.exe", "xcbuild.exe")]
+        [TestCase(@"subFolder\xcbuild.exe", @"subFolder\xcbuild.exe")]
+        [TestCase(@"..\subFolder\xcbuild.exe", @"subFolder\xcbuild.exe")]
+        public void TestSetXcBuildPath(string path, string expectedSubPath)
+        {
+            var cakeContext = Substitute.For<ICakeContext>();
+            cakeContext.SetXcBuildPath(path);
+            if (path.Contains(".."))
```

Comment:
this test is weird... 
what's the intended behavior when path.Contains("..")?
could it return a new random string?